src/librustc/lib.rs

```diff
@@ -86,6 +88,7 @@ extern crate graphviz;
 extern crate libc;
 extern crate polonius_engine;
 extern crate rustc_target;
+#[macro_use] extern crate rustc_macros;
```

Comment:
FWIW wherever possible it'd be great to stop using `#[macro_use]` for new macro-based code